storm/tests/test_storm.py

```diff
@@ -0,0 +1,644 @@
+        ('Bolt2', (2, 0.015, 0.010, 0.000, 0, 3153, 0, 3153, 0, 2, 1E10, 0, 0, 0)),
+        ('Bolt3', (3, 0.009, 0.003, 0.000, 0, 4704, 0, 4704, 0, 3, 1E10, 0, 0, 0)),
+        ('Bolt4', (4, 0.001, 291.756, 0.000, 0, 218808, 101607, 110946, 101607, 4, 1E10, 0, 0, 0)),
+        ('Bolt5', (2, 0.001, 1014.634, 0.000, 0, 208890, 17, 104445, 17, 2, 1E10, 0, 0, 0)),
+        ('Bolt6', (3, 0.010, 0.005, 0.000, 0, 4705, 0, 4705, 0, 3, 1E10, 0, 0, 0))
+    ]:
+        test_tags = storm_version_tags + env_tags + topology_tags + ['bolt:{}'.format(name)]
+        for i, metric_name in enumerate([
+            'tasks', 'executeLatency', 'processLatency', 'capacity', 'failed', 'acked', 'transferred', 'executed',
+            'emitted', 'executors', 'errorLapsedSecs', 'requestedMemOnHeap', 'requestedCpu', 'requestedMemOffHeap'
+        ]):
+            aggregator.assert_metric(
+                'storm.bolt.last_60.{}'.format(metric_name),
+                value=values[i],
+                tags=test_tags,
+                count=1
+            )
+
+    # Spout Stats
+    for name, values in [
+        ('source', (8, 285.950, 0, 104673, 104673, 104673, 8, 38737, 0, 0, 0)),
+    ]:
+        test_tags = storm_version_tags + topology_tags + env_tags + ['spout:{}'.format(name)]
+        for i, metric_name in enumerate([
+            'tasks', 'completeLatency', 'failed', 'acked', 'transferred', 'emitted', 'executors', 'errorLapsedSecs',
+            'requestedMemOffHeap', 'requestedCpu', 'requestedMemOnHeap'
+        ]):
+            aggregator.assert_metric(
+                'storm.spout.last_60.{}'.format(metric_name),
+                value=values[i],
+                tags=test_tags,
+                count=1
+            )
+
+    # # Topology Metrics
+    # metric_cases = (
+    #     # Topology Metrics By Bolt
+    #     ('storm.topologyStats.metrics.bolts.last_60.transferred', 0.0,
+    #         storm_version_tags + topology_tags + env_tags + ['bolts:count', 'stream:__system']),
+    # )
+    # for m in ['acked', 'complete_ms_avg', 'emitted', 'transferred']:
+    #     aggregator.assert_metric(
+    #         'storm.topologyStats.metrics.spouts.last_60.{}'.format(m),
+    #         at_least=1
+    #     )
+
+    # for m in ['acked', 'emitted', 'executed', 'executed_ms_avg', 'process_ms_avg', 'transferred']:
+    #     aggregator.assert_metric(
+    #         'storm.metrics.bolts.last_60.{}'.format(m),
+    #         at_least=1
+    #     )
+
+    # for case in metric_cases:
+    #     aggregator.assert_metric(case[0], value=case[1], tags=case[2], count=1)
+
+    # # Raises when COVERAGE=true and coverage < 100%
+    # aggregator.assert_all_metrics_covered()
+
+
+@pytest.mark.integration
+def test_integration_with_ci_cluster(dd_environment, aggregator):
+    check = StormCheck(STORM_CHECK_INTEGRATION_CONFIG, {})
+
+    # run your actual tests...
+    check.check(STORM_CHECK_INTEGRATION_CONFIG['instances'][0])
+
```

Comment:
If you yield the right thing from dd_environment, you can use it directly here instead of the global.